ComfyUI/utils/lora_merger.py

```
@@ -0,0 +1,187 @@
+
+    Returns:
+        bool: True if merge was successful, False otherwise
+    """
+    try:
+        # Validate input files
+        if not validate_safetensors_file(base_path):
+            return False
+        if not validate_safetensors_file(lora_path):
+            return False
+
+        logger.info(f"Starting LoRA merge using ComfyUI infrastructure:")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)